grc/core/params/param.py

```diff
@@ -178,6 +178,10 @@ def evaluate(self):
         # ID and Enum types (not evaled)
         #########################
         if dtype in ('id', 'stream_id') or self.is_enum():
+            if self.options.attributes:
+                expr = collections.UserString(expr)
```

Comment:
since we're not trying to override any methods (only add attributes), defining a new class `mstr = type('mstr', (str,), {})` would have been my first choice,  but this works as well.